.storybook/preview.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+import { previewTheme } from './gardenTheme';
+import { ThemeProvider, DEFAULT_THEME } from '../packages/theming/src';
+
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*', layout: 'centered' },
+  docs: {
+    theme: previewTheme
+  }
+};
+
+const GlobalPreviewStyling = createGlobalStyle`
+  body {
+    font-family: ${p => p.theme.fonts.system};
+    font-size: ${p => p.theme.fontSizes.md};
+  }
+`;
+
+const StyledExampleWrapper = styled.div`
+  direction: ${p => (p.theme.rtl ? 'rtl' : 'ltr')};
+  padding: ${p => p.theme.space.xl};
+`;
+
+const withThemeProvider = (Story, context) => {
+  const rtl = context.globals.locale === 'rtl';
+
+  if (context.globals.bedrock === 'enabled') {
+    document.querySelector('link[href$="bedrock/index.css"]').removeAttribute('disabled');
+  } else {
+    document.querySelector('link[href$="bedrock/index.css"]').setAttribute('disabled', true);
+  }
+
+  return (
+    <ThemeProvider theme={{ ...DEFAULT_THEME, rtl }}>
+      <GlobalPreviewStyling />
+      <StyledExampleWrapper>
+        <Story />
+      </StyledExampleWrapper>
+    </ThemeProvider>
+  );
+};
+
+export const decorators = [withThemeProvider];
+
+export const globalTypes = {
+  locale: {
+    name: 'direction',
+    description: 'Locale direction',
+    defaultValue: 'ltr',
+    toolbar: {
+      icon: 'globe',
+      items: [
+        { value: 'ltr', title: 'LTR' },
+        { value: 'rtl', title: 'RTL' }
+      ]
+    }
+  },
+  bedrock: {
+    name: 'bedrock',
+    description: 'CSS Bedrock',
+    defaultValue: 'disabled',
+    toolbar: {
+      icon: 'paintbrush',
+      items: [
+        { value: 'disabled', title: 'Disabled' },
+        { value: 'enabled', title: 'Enabled' }
+      ]
+    }
+  }
```

Comment:
Can these be made to work like the "Grid" toggle rather than choosing from a menu?

Reply:
Looking through their docs I haven't been able to find anything about different global toolbar types. Since it's an essential plugin it seems they have a deeper integration than we do in `preview.js`. I'll keep looking through their docs though.

Reply:
Does this shed any light? https://github.com/storybookjs/storybook/blob/cf7c1c33dfe05e4dbe93f6121d42d058e5db3825/addons/backgrounds/src/containers/BackgroundSelector.tsx#L189-L209

Reply:
Without a fully-custom addon I'm not sure we will be able to replicate this. Feels like it could be a good follow-on before we would make the global cut.

Reply:
Ok, let's capture the task and see if we can make something of it. Just having the functionality is fine for now.